src/clients/HubPoolClient.ts

```diff
@@ -271,6 +272,23 @@ export class HubPoolClient extends BaseAbstractClient {
     return this.getL2TokenForL1TokenAtBlock(l1Token, l2ChainId, deposit.quoteBlockNumber);
   }
 
+  l2TokensAreEquivalent(
+    tokenA: string,
+    chainIdA: number,
+    tokenB: string,
+    chainIdB: number,
+    latestHubBlock = Number.MAX_SAFE_INTEGER
+  ): boolean {
+    try {
+      return (
+        this.getL1TokenForL2TokenAtBlock(tokenA, chainIdA, latestHubBlock) ===
+        this.getL1TokenForL2TokenAtBlock(tokenB, chainIdB, latestHubBlock)
+      );
+    } catch (err) {
```

Comment:
Convenience method used in BundleDataClient to validate a slow fill request, but I don't love how it hides the error.

Reply:
This is partially duplicating the upstream function `HubPoolClient.tokensAreEquivalent()` (but not completely). Is there a reason it can't be used?

https://github.com/across-protocol/sdk-v2/blob/9a2f0f464936dbc28ce0552ad36922fc3d9ede59/src/clients/HubPoolClient.ts#L512

Reply:
oops didn't see this. will replace